packages/api-file-manager/src/plugins/crud/settings.crud.ts

```diff
@@ -36,13 +36,22 @@ export default (context: FileManagerContext): SettingsCRUD => {
     const PK_SETTINGS = () => `T#${security.getTenant().id}#FM#SETTINGS`;
     const SK_SETTINGS = () => `default`;
 
+    const cache = {};
```

Comment:
Let's remove this caching. I think it's really not that needed, and who knows if down the road could cause some issues / inconsistencies for the user (we don't have cache invalidation logic here).

Reply:
Sure. I'm always up for less code :)